### DISCUSSION TITLE
Custom branding icons

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#129] [Polls]
Hey, what do you think about custom branding icons? Do you use them? Should we remove (some of) them or add even more? 

They increase the size of the patches but is it noticeable?

**Comment by anddea** [comment]
I see some of us want to remove some icons. I think we have to discuss which one shouldn't be here.

**Reply by STEK1337** [reply]
I don't see any reason to remove some icons, as long as the `Custom branding icon YouTube` patch is enabled anyone can implement their icon

**Reply by tooruu** [reply]
MMT icon is as good as it gets IMO. Others I don't like as much.
As for custom heading, I use ReVanced heading. Hoping to see it copied over.

**Reply by MtndaleRedneck** [reply]
I use Revancify Red

**Reply by fuengfah** [reply]
Do you have a list of all valid values for custom icon?

**Reply by anddea** [reply]
> Do you have a list of all valid values for custom icon?

You can see all of them in the Manager. Although it doesn't show a preview of icons.

**Reply by ghost** [reply]
> MMT icon is as good as it gets IMO. Others I don't like as much.
> As for custom heading, I use ReVanced heading. Hoping to see it copied over.

Yeah the MMT icon looks cool but the other two (Revancify Blue & Red) doesn't have that modern feel. New flat icons could be added.

**Reply by ghost** [reply]
> Do you have a list of all valid values for custom icon?

Theres only three I really know of (picture below):

1. mmt
2. revancify_blue
3. revancify_red

![Screenshot_2024-04-04-22-58-47-076_com.brave.browser-edit.jpg](https://github.com/anddea/revanced-patches/assets/86847643/457709c5-d5f5-4f12-a8b8-ff6bfbe96a01)

Image Source: https://github.com/ReVanced-Extended-Community/Patches-Documentation?tab=readme-ov-file#youtube-1

**Comment by MarkoAPK** [comment]
Hi friend. I created acc on github ONLY for vote and comment this thread!

pls dont remove MMT icon,add AFN icon its very nice. Me and my friends alwayes chose them

**Reply by anddea** [reply]
Probably you already know, AFN icons was added in previous releases.

**Reply by MarkoAPK** [reply]
big thx !!!
